FreeRTOS/Test/Target/tests/smp/schedule_equal_priority/schedule_equal_priority.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * FreeRTOS V202212.00
+ * Copyright (C) 2022 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ *
+ * https://www.FreeRTOS.org
+ * https://github.com/FreeRTOS
+ *
+ */
+
+/**
+ * @file schedule_equal_priority.c
+ * @brief The scheduler shall schedule tasks of equal priority in a round robin fashion.
+ *
+ * Procedure:
+ *   - Create ( num of cores + 1 ) tasks ( T0~Tn ). Priority T0 = T1 = ... = Tn-1 = Tn.
+ *   - All three tasks are running in busy loop
+ * Expected:
+ *   - All three tasks get a chance to run. The test doesn't currently strictly
+ *     validate the round-robin nature of the scheduler as other system tasks,
+ *     possibly target specific come and go complicating on-target validation.
```

Comment:
Equal priority tasks are scheduled in a round robin fashion when configUSE_TIME_SLICING is set to 1. Verify that all the created equal priority task get chance to run.
```suggestion
 *   - Equal priority tasks are scheduled in a round robin fashion when configUSE_TIME_SLICING is set to 1. Verify that all the created equal priority tasks get chance to run.
```
Please also fix the indent.